build/storage/tests/it/ptf_tests/README.md

```diff
@@ -0,0 +1,8 @@
+# Description
+PTF tests are integration tests based on PTF library. This library was chosen for the matter of unification of testing framework between development teams. 
+
+# Run tests
+1. Firstly, create python environment. To do this, run command `bash create_python_environment.sh` within storage/tests/it/ptf_scripts directory.
```

Comment:
Simplify the description. Numbered list already prescribes order.
E.g.
1. Create Python environment by issuing `bash ...` within ...

Reply:
corrected